### PR TITLE
Feat : copy to clipboard btn added

### PR DESCRIPTION
**Pull request by rohitd99**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,11 +5,18 @@
   "version": "1.6.0",
   "license": "MIT",
   "dependencies": {
+    "@gatsbyjs/reach-router": "^2.0.1",
```
Comment (thecodedose) [inline package.json:8]: Why did you add so many new dependencies?

File: src/styles/copy_btn.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    font-weight: 500;
+    padding: 3px 3px 2px;
+    border-radius: 3px;
+    color: rgba(255, 255, 255, 0.88);
+}
+
+.gatsby-remark-prismjs-copy-button:hover {
+    background-color: #ff84ca;
+    color: #2a2a2a;
+    filter: drop-shadow(2px 2px white);
```
Comment (thecodedose) [inline src/styles/copy_btn.css:29]:
```suggestion
```

File: src/styles/copy_btn.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    font-size: 10px;
+    font-weight: 500;
+    padding: 3px 3px 2px;
+    border-radius: 3px;
+    color: rgba(255, 255, 255, 0.88);
+}
+
+.gatsby-remark-prismjs-copy-button:hover {
+    background-color: #ff84ca;
+    color: #2a2a2a;
```
Comment (thecodedose) [inline src/styles/copy_btn.css:28]: Can you add a button border on hover instead? just a white border

**Review by thecodedose — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (rohitd99) [reply package.json:8]: Initially while setting up the project , I had run into these module not found errors and so I had installed them. I kind of realised that it would be an issue, let me try and setup things again.

Comment (rohitd99) [reply package.json]: I've used npm instead of pnpm , so there's no module not found errors now. They had arised due to webpack being unable to resolve the dependencies. There is a [plugin](https://www.gatsbyjs.com/plugins/gatsby-plugin-pnpm/) to solve this but I used npm rather than going with the plugin.

File: src/styles/copy_btn.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    border-radius: 3px;
+    border: 1px solid transparent;
+    transition: border 0.2s ease-in-out;
+    color: rgba(255, 255, 255, 0.88);
+}
+
+.gatsby-remark-prismjs-copy-button:hover {
+    border: 1px solid white;
+    background-color: transparent;
+}
```
Comment (thecodedose) [inline src/styles/copy_btn.css:31]:
```suggestion
}

```

**Review by thecodedose — APPROVED** [review]
